node/src/BaseClient.ts

```diff
@@ -1491,6 +1493,60 @@ export class BaseClient {
         );
     }
 
+    /** Returns a random element from the set value stored at `key`.
+     * See https://valkey.io/commands/srandmember for more details.
+     *
+     * @param key - The key from which to retrieve the set member.
+     * @returns a random element from the set, or null if `key` does not exist.
```

Comment:
```suggestion
     * @returns A random element from the set, or null if `key` does not exist.
```
Should we capitalize? If not, please ignore